test/e2e/localvolumeset_test.go

```diff
@@ -0,0 +1,334 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+
+	localv1 "github.com/openshift/local-storage-operator/pkg/apis/local/v1"
+	localv1alpha1 "github.com/openshift/local-storage-operator/pkg/apis/local/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	labelNodeRoleWorker = "node-role.kubernetes.io/worker"
+)
+
+func LocalVolumeSetTest(ctx *framework.Context, cleanupFuncs *[]cleanupFn) func(*testing.T) {
```

Comment:
Why do we need gomega? Most of the e2e worked fine without it. Coming from k8s - i would rather avoid it if I can.

Reply:
I used it because gomega.Eventually is very useful. 
Not using the whole ginkgo+gomega, this is just an assertion utility on a `*testing.T`.

Reply:
hmm.. kinda a nit but nonetheless, I do not see value in bringing yet another dependency. we already had - https://github.com/stretchr/testify which provides a decent collection of assertions that we can use without having to import yet another matcher/assertion library.

In terms of style and coherency of code - it makes maintenance easier if one style is followed throughout the code. I would probably remove gomega completely if I were you, but that just may be me.